docs/src/dev-contributors.md

```
@@ -0,0 +1,145 @@
+<script>
+  document.addEventListener('DOMContentLoaded', async () => {
+    try {
+      // Fetch contributors from GitHub API
+      const response = await fetch('https://api.github.com/repos/truefoundry/elasti/contributors');
+      
+      if (!response.ok) {
+        throw new Error(`GitHub API responded with status: ${response.status}`);
+      }
+      
+      const contributors = await response.json();
+      
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 6

(MD009, no-trailing-spaces)